env/variables.tf

```diff
@@ -0,0 +1,44 @@
+variable "aws_key_path" {}
+variable "aws_key_name" {}
+
+variable "prefix" {
+    description = "Prefix all named resources"
+    default = "<env name>"
```

Comment:
does AWS allow to use a space, in case the default value is being used?

Reply:
It's only names and tags so I would guess probably yes.

Reply:
If you did do that you'd end up with a bunch of stuff called <env name>-security-group... in the AWS console and then if you set the variable and ran terraform apply again it would just rename it all.